src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const splitBySentence = sourceString.split(';');
+
+  for (let i = 0; i < splitBySentence.length; ++i) {
+    const current = splitBySentence[i].trim();
+
+    if (current) {
+      const [key, value] = current.split(':');
+
+      if (key && value) {
+        result[key.trim()] = value.trim();
+      }
+    }
+  }
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. You can use the `split` method to create an array of style declarations and then `map` to transform each declaration into a key-value pair, followed by `reduce` to accumulate them into an object.